test/test_other.py

```diff
@@ -15141,3 +15141,15 @@ def test_extra_struct_info(self):
   def test_fs_writev_partial_write(self):
     self.set_setting('FORCE_FILESYSTEM')
     self.do_run_in_out_file_test('fs/test_writev_partial_write.c')
+
+  @requires_v8
+  @parameterized({
+    '': [[]],
+    'O2': [['-O2']]
```

Comment:
Maybe `-O3` for the most coverage?

Reply:
Do you recommend in addition to or in replace of O2?